environment-windows.yml

```diff
@@ -11,5 +11,5 @@ dependencies:
   - pip:
     - cpplint==1.4.4
     - yapf==0.27.0
-    - win32event
+    - pypiwin32
```

Comment:
This should probably be `pywin32`, not sure `pypiwin32` is the official package.